File: overview/rev1/guides/examples/java/credit-soft-descriptor.java
// bankAccountHref is the stored href for the bank account
BankAccount bankAccount = new BankAccount(bankAccountHref);

HashMap<String, Object> payload = new HashMap<String, Object>();
payload.put("amount", 100000);
payload.put("description", "Payout for order #1111");
payload.put("appears_on_statement_as", "GoodCo #1111");

try {
    Credit credit = bankAccount.credit(payload);
}
catch (HTTPError e) {}

File: overview/rev1/guides/examples/csharp/credit-soft-descriptor.cs
// bank_account_href is the stored href for the BankAccount
BankAccount bankAccount = BankAccount.Fetch(bank_account_href);
Dictionary<string, object> creditPayload = new Dictionary<string, object>();
creditPayload.Add("amount", 5000);
creditPayload.Add("description", "Payout for order #1111");
creditPayload.Add("appears_on_statement_as", "GoodCo #1111");
Credit credit = bankAccount.Credit(creditPayload);